modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -19383,6 +19384,7 @@ this-value-cannot-be-less-than-1=This value cannot be less than 1.
 this-value-is-used-for-alt-text=This value is used as ALT text.
 this-version=This version
 this-version-will-be-published-when-x-is-published-to-live=This version will be published when <em>{0}</em> is published to live.
+this-visualization-mode-will-not-be-shown-until-you-assign-at-least-one-field-to-a-card-element=This visualization mode won't be shown until you assign at least one field to a card element.
```

Comment:
[nit] `won't` should probably be `will not` in both values.

Reply:
Right!

Reply:
I'm not sure either. Not blocking for PR, in any case.